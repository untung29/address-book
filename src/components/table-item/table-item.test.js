import React from "react";
import renderer from "react-test-renderer";
import TableItem from "./table-item.component";

it("<TableItem /> Snapshot Testing", () => {
  const TableItemTest = renderer.create(<TableItem item="Untung" />).toJSON();
  expect(TableItemTest).toMatchSnapshot();
});
