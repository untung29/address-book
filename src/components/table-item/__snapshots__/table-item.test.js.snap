// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TableItem /> Snapshot Testing 1`] = `
<td>
  Untung
</td>
`;
